src/components/send/SendHeader.js

```diff
@@ -125,47 +124,30 @@ export default function SendHeader({
   const isPreExistingContact = (contact?.nickname?.length || 0) > 0;
 
   const name =
-    removeFirstEmojiFromString(
-      userWallet?.label || contact?.nickname || nickname
-    ) ||
+    userWallet?.label ||
+    contact?.nickname ||
+    nickname ||
     userWallet?.ens ||
     contact?.ens ||
     recipient;
 
   const handleNavigateToContact = useCallback(() => {
-    let nickname = profilesEnabled
-      ? !isHexString(recipient)
-        ? recipient
-        : null
-      : recipient;
-    let color = '';
-    if (!profilesEnabled) {
-      color = contact?.color;
-      if (color !== 0 && !color) {
-        const emoji = profileUtils.addressHashedEmoji(hexAddress);
-        color = profileUtils.addressHashedColorIndex(hexAddress) || 0;
-        nickname = isHexString(recipient) ? emoji : `${emoji} ${recipient}`;
-      }
-    }
-
     android && Keyboard.dismiss();
     navigate(Routes.MODAL_SCREEN, {
       additionalPadding: true,
       address: hexAddress,
-      color,
-      contact,
+      contactNickname: userWallet?.label ?? contact?.nickname,
```

Comment:
`userWallet` is coming from the imported wallets, i think it makes sense to respect the name of the wallet, but not sure if we're doing the same for other places where we use `contact?.nickname`

Reply:
should we make it the standard from the contact state? ... this could potentially add some edge cases like what happens when a wallet name is changed, then we'll need to update the contact nickname of the contact, WDYT?

Reply:
https://github.com/rainbow-me/rainbow/pull/3464/files#diff-4de07ac1507934166c3e0f8a48da8c3a3ef59474892b1341736efb515cc21574R218 here we don't use the `userWallet?.label` for example